static/documents/tradetrust/v2/ebl-goerli.json

```diff
@@ -0,0 +1,56 @@
+{
+  "version": "https://schema.openattestation.com/2.0/schema.json",
+  "data": {
+    "name": "02736f77-d18a-4ce4-867a-5e680337aedd:string:TradeTrust Bill of Lading v2",
+    "shipper": {
+      "address": {
+        "street": "31f5f70e-8d01-42d0-afad-45e7bfc5f959:string:One North",
+        "country": "4e8c79ab-922d-44cc-8d81-7a3c4f7faf9a:string:Singapore"
+      },
+      "name": "199a9550-2a6d-4edd-a8f0-b067ed290ac4:string:Demo Shipper"
+    },
+    "consignee": {
+      "name": "b963cd9b-1273-494a-a5d5-2385255b48a0:string:Demo Consignee"
+    },
+    "notifyParty": {
+      "name": "5caae3b7-9e3c-4f0c-b65b-780594344c1f:string:Demo Notify"
+    },
+    "$template": {
+      "type": "c0644a87-03dd-4ba3-a6bd-8005eab66569:string:EMBEDDED_RENDERER",
+      "name": "9ebf8a4d-4664-40e1-9749-e7d0270170f8:string:BILL_OF_LADING",
+      "url": "61e2d3d8-8ec0-402f-84a2-7acbf78b27d1:string:https://generic-templates.tradetrust.io"
+    },
+    "issuers": [
+      {
+        "identityProof": {
+          "type": "24917ac0-3a24-4dce-a80d-49283cc7ba6c:string:DNS-TXT",
+          "location": "9bda2a89-9cf1-4507-8b5a-b0a5fd4a0f57:string:demo-tradetrust.openattestation.com"
+        },
+        "name": "ca252a45-d6ba-420c-96bc-8e09cdd4c2c2:string:Demo token registry",
+        "tokenRegistry": "43b6251d-fd09-428b-8f87-9d1c8a3ace29:string:0x921dC7cEF00155ac3A33f04DA7395324d7809757"
+      }
+    ],
+    "blNumber": "2c66c099-c204-43ed-b17b-17249e969cdf:string:123",
+    "vessel": "67b52fc4-d242-46d1-b820-892e343bdbb1:string:1",
+    "voyageNo": "22538e20-cb3c-446e-b681-6c38a1fa231b:string:100",
+    "portOfLoading": "6046ee16-8963-47dc-9f7f-dab8fbe45567:string:Singapore Port",
+    "portOfDischarge": "2f332257-b62f-4f63-82a1-456740e9bc16:string:China Port",
+    "placeOfReceipt": "ec26e3c1-2e1a-4f0b-8341-36f4787aa0de:string:Beijing",
+    "placeOfDelivery": "cc7b5440-dec3-4f59-95c1-68ee3b6232a7:string:Singapore",
+    "links": {
+      "self": {
+        "href": "04e8165a-7051-41ac-8b11-126d6c9979fe:string:https://action.openattestation.com?q=%7B%22type%22%3A%22DOCUMENT%22%2C%22payload%22%3A%7B%22uri%22%3A%22https%3A%2F%2Fgallery.openattestation.com%2Fstatic%2Fdocuments%2Ftradetrust%2Fv2%2Febl-goerli.json%22%2C%22permittedActions%22%3A%5B%22VIEW%22%5D%2C%22redirect%22%3A%22https%3A%2F%2Fdev.tradetrust.io%22%2C%22chainId%22%3A%20%225%22%7D%7D"
+      }
+    },
+    "network": {
+      "chain": "ab094b05-fa90-462f-8b50-ef033a8e83f4:string:ETH",
+      "chainId": "2a490e84-c36b-40c1-a7e9-4069fba6b901:string:5"
+    }
```

Comment:
Ahh... Notice that this wasn't in the old ropsten version of the file. Cool!👍

Reply:
yeah this 1 just incase, say if next time we don't need the workaround to pass chainId